
export const signupController = async()=>{
    try{

    }catch(error){
        console.log("Error :",error);
        return res.status(500).json({ message:"Internal server Error"});
    }
}

export const signinController = async()=>{
    try{

    }catch(error){
        console.log("Error :",error);
        return res.status(500).json({ message:"Internal server Error"});
    }
}

export const signoutController = async()=>{
    try{

    }catch(error){
        console.log("Error :",error);
        return res.status(500).json({ message:"Internal server Error"});
    }
}


